modules/ocf/facts.d/puppetdb-running

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -uo pipefail
+fqdn=$(hostname -f)
+
+# Due to differences between the puppet and puppet-agent packages, the SSL directory may be in /etc/puppetlabs/puppet or /var/lib/puppet
+puppet_ssl_path=$(puppet config print ssldir)
+
+curl 'https://puppetdb:8081/pdb/query/v4/nodes' -s --tlsv1 --cacert $puppet_ssl_path/certs/ca.pem --cert $puppet_ssl_path/certs/$fqdn.pem --key $puppet_ssl_path/private_keys/$fqdn.pem > /dev/null
```

Comment:
I think replacing this with something like `https://puppetdb:8081/pdb/meta/v1/version` or `https://puppetdb:8081/pdb/meta/v1/server-time` would be better, since then it doesn't have to return the details for all the nodes and instead only checks that the server is up. Also if you can wrap this to multiple lines it would probably be good, since it's quite long as written like this.

Reply:
Also, the variables in this line ought to be quoted. Even though they aren't expected to contain a space, it's a good habit to get into.